static/css/mozsearch.css

```diff
@@ -388,7 +394,7 @@ body {
 }
 
 .content {
-    padding: 1rem 2rem;
+    padding: 0.5rem 0rem;
```

Comment:
This removes the padding on the dir listing pages too (e.g. https://dev.searchfox.org/mozilla-central/source) which seems slightly undesirable. At least I prefer those listing pages with the padding and it looks weird if the the entries are shoved up against the left edge of the browser. But I don't feel strongly about it, I'm sure I'll get used to it.

Reply:
I think it makes sense to put the padding back by default.  I'll augment the content node (`<div id="content class="content" ...>` so that it also provides a class indicating what the type of the content is.  So the `.content` can go back to the way it was and `.content.source-listing` can opt for decreased padding so we can get the "full bleed" edge-to-edge look for the sticky headers.

Another possibility would be to just use negative margins for the source listing case, but that seems like adding extra conceptually complexity to what's otherwise a reasonably straightforward table-ish thing.